src/components/application_manager/src/commands/command_impl.cc

```diff
@@ -214,5 +214,21 @@ DEPRECATED void CommandImpl::ReplaceHMIByMobileAppId(
   }
 }
 
+// static
```

Comment:
@APipko What is this comment here for?

Reply:
@AByzhynar This comment is the notice about static method, like for another static methods in sdl_core

Reply:
@APipko as for me '// static' is redundant comment.

Reply:
@AByzhynar fixed in 6057cfc3c4a9f31c19569f7f6b335e0077921d65